rules/test/lldb/lldb_sim_runner.py

```diff
@@ -209,8 +209,7 @@ def GetStartedAppPid(self):
     def Fail(self):
         traceback.print_exc()
         logger.error("FAIL")
-        with ctxlock:
-            self.status = -1
+        self.status = -1
```

Comment:
Unrelated but `ctxlock` is undefined so when a lldb test would fail this would mask the actual error.